examples/angular2/package.json

```diff
@@ -1,16 +1,32 @@
 {
   "private": true,
   "scripts": {
-    "dev": "tsc --watch"
-  },
-  "devDependencies": {
-    "typescript": "1.7.5"
+    "start": "tsc && concurrently \"tsc -w\" \"lite-server\" ",
+    "typings": "typings"
   },
   "dependencies": {
-    "angular2": "2.0.0-beta.0",
-    "systemjs": "0.19.6",
-    "rxjs": "5.0.0-beta.0",
+    "@angular/common": "~2.0.2",
+    "@angular/compiler": "~2.0.2",
+    "@angular/core": "~2.0.2",
+    "@angular/forms": "~2.0.2",
+    "@angular/http": "~2.0.2",
+    "@angular/platform-browser": "~2.0.2",
+    "@angular/platform-browser-dynamic": "~2.0.2",
+    "@angular/router": "~3.0.2",
+
+    "systemjs": "0.19.39",
+    "core-js": "^2.4.1",
+    "reflect-metadata": "^0.1.8",
+    "rxjs": "5.0.0-beta.12",
+    "zone.js": "^0.6.25",
```

Comment:
Per comment in another commit, we should be able to drop rxjs as a dep in the package.json entirely.